src/components/Budget/BudgetLineitem/__test__/BudgetLineItem.test.tsx

```diff
@@ -208,25 +208,30 @@ beforeEach(() => {
 	});
 });
 
-const inputIds = budgetLineItemInputFields;
+const budgetLineItemInputIds = budgetLineItemInputFields;
 
 describe("Budget Line Item Dialog tests", () => {
```

Comment:
What functionality are we exactly testing here?

Reply:
I have mentioned the names of the input field and the key to access their values from  an object called intialFormValue in a seprate file. Then i loop through all the input fields and insert the value and check if the values are modified and the input field has the correct value, then I check if the submit button got enabled, then I check if the request got submitted. All of these test are releated to each other,

Reply:
I have mentioned the id of the input element in a seprate file and the mocks in seprate file, and initial values of a form is mentioned seprately in every test file.